internal/authtoken/authtoken.go

```diff
@@ -97,14 +105,20 @@ func newAuthTokenId() (string, error) {
 	return id, nil
 }
 
-// newAuthToken generates a token with a version prefix.
-func newAuthToken() (string, error) {
+// newAuthToken generates a new in-memory token.  No options are currently
+// supported.
+func newAuthToken(_ ...Option) (*AuthToken, error) {
 	const op = "authtoken.newAuthToken"
 	token, err := base62.Random(tokenLength)
 	if err != nil {
-		return "", errors.Wrap(err, op, errors.WithCode(errors.Io))
+		return nil, errors.Wrap(err, op, errors.WithCode(errors.Io))
 	}
-	return fmt.Sprintf("%s%s", TokenValueVersionPrefix, token), nil
+
+	return &AuthToken{
+		AuthToken: &store.AuthToken{
+			Token: token,
```

Comment:
Is there a reason why you removed the auth token version prefix?

Reply:
That was just an oversight during the refactor.  Ty for catching this!